userProfile/migrations/0002_remove_userprofile_id_userprofile_profile_pic_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.0.3 on 2022-03-25 09:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Can you use a shorter and more meaningful file name? Why do you need 2 migration files?